microfactory/microfactory.go

```diff
@@ -69,6 +69,7 @@ var (
 
 	goToolDir = filepath.Join(runtime.GOROOT(), "pkg", "tool", runtime.GOOS+"_"+runtime.GOARCH)
 	goVersion = findGoVersion()
+	isGo18    = strings.Contains(goVersion, "go1.8")
```

Comment:
A possible hack that may be more accurate, check if go/build.Default.ReleaseTags contains "go1.8"?